modules/example/src/smithy4s/example/package.scala

```diff
@@ -4,11 +4,26 @@ package object example {
   val NAMESPACE: String = "smithy4s.example"
 
   type StreamedObjects[F[_]] = smithy4s.Monadic[StreamedObjectsGen, F]
-  val StreamedObjects : smithy4s.Service[StreamedObjectsGen, StreamedObjectsOperation] = StreamedObjectsGen
+  object StreamedObjects extends smithy4s.Service.Provider[StreamedObjectsGen, StreamedObjectsOperation] {
```

Comment:
I think turning it into an object might help metals ?

Reply:
indeed, it should help autoimport!

Reply:
aaaaaand it does.